collect/dwr/cdec/queries.py

```diff
@@ -162,10 +161,14 @@ def get_raw_station_csv(station, start, end, sensors=[], duration='', filename='
                      header=0, 
                      parse_dates=True, 
                      index_col=4, 
-                     na_values=['m', '---', ' ', 'ART', 'BRT', -9999, -9998, -9997],
+                     na_values=['m', '---', ' ', -9999, -9998, -9997],
                      float_precision='high',
                      dtype=default_data_types)
-    df['DATE TIME'] = df.index
+
+    #report if the data is BRT or ART (below/above rating table)
+    df.loc[df['VALUE'] == 'BRT', 'RATING_FLAG'] = 'BRT'
+    df.loc[df['VALUE'] == 'ART', 'RATING_FLAG'] = 'ART'
+    df['VALUE'] = df['VALUE'].replace({'BRT': None, 'ART': None}).astype(float)
```

Comment:
No warnings for overwriting the series with a copy of the series here? (L171)

Reply:
no warnings